docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -130,16 +130,16 @@ endif::[]
 
 .Procedure
 
-. In your IDE, create a new file. Save the file with the name `.env`, at the root level of the `reactive-example` directory for the cloned repository.
+. In your IDE, create a new file. Save the file with the name `rhoas.env`, at the root level of the `reactive-example` directory for the cloned repository.
 
-. In the `.env` file, add the lines shown in the example. These lines set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
+. In the `rhoas.env` file, add the lines shown in the example. These lines set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
 +
-.Setting environment variables in the .env file
+.Setting environment variables in the rhoas.env file
 [source,subs="+quotes"]
```

Comment:
I think we can consider changing this example to use RHOAS CLI context in the future

Reply:
SOunds good :+1: